src/screens/Assets/WalletView.js

```diff
@@ -313,11 +313,6 @@ class WalletView extends React.Component<Props, State> {
     });
   };
 
-  getStickyIndices = () => {
-    const { showDeploySmartWallet } = this.props;
-    return !this.isInSearchAndFocus() && !this.shouldBlockAssetsView() && showDeploySmartWallet ? [2] : [1];
```

Comment:
Are you sure it's safe to delete the `? [2] : [1]` and replace it with `[1]` for all cases?

Reply:
Yes, I've wrapped upper components so they are all now considered to be first child ([0]), so no matter what, we can always make second one ([1]) sticky (a group of SearchBar and Tabs)